moped-database/metadata/tables.yaml

```diff
@@ -1358,79 +1358,63 @@
           table:
             name: moped_subphases
             schema: public
-  insert_permissions:
```

Comment:
Revokes insert permissions for all roles on the `moped_phases` table. this is an issue that Frank picked up on—and needs to be addresses on other lookup tables as well.